Makefile

```diff
@@ -33,15 +33,17 @@ INSTALLED_MAN_TARGETS = $(addprefix $(PREFIX)/share/man/man1/, $(MAN_TARGETS))
 
 # source, dependency and build definitions
 DEPDIR = .d
-$(shell install -d $(DEPDIR))
 MAKEDEPEND = echo "$@: $$(go list -f '{{ join .Deps "\n" }}' $< | awk '/github/ { gsub(/^github.com\/[a-z]*\/[a-z]*\//, ""); printf $$0"/*.go " }')" > $(DEPDIR)/$@.d
 
 $(DEPDIR)/%.d: ;
 .PRECIOUS: $(DEPDIR)/%.d
 
+$(DEPDIR):
+	install -d $@
+
```

Comment:
Ah sorry, I should have talked it through.  I hit issues with the bare `$(shell install -d $(DEPDIR))` in combination with explicit `make all`.